### DISCUSSION TITLE
RDP 2 Winodws

### DISCUSSION THREAD

**Discussion by BeybarsKZ** [homeassistant-apps/app-cloudflared#705] [Q&A]
Hi. I have set up Clodflare addon for Home Assistant and it works great! Can you tell me if it is possible to use addon to connect to windows station via RDP?
I tried to do it like this, but I could not connect to windows station:
- hostname: rdp.mydomain.com
service: rdp://192.168.1.137:3389

**Comment by brenner-tobias** [comment]
My understanding is that this works, there is no restriction from the add-on. I suggest following the [remote tunnel set-up](https://github.com/brenner-tobias/addon-cloudflared/wiki/How-tos#how-to-configure-remote-tunnels) so you have the full flexibility in the configuration in the Cloudflared Zero Trust Dashboard. Since it looks like with RDP there is more to it then just exposing it via the tunnel, please follow [this guide](https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/use-cases/rdp/).
Since this is not add-on related, but a general Cloudflared topic, I suggest to open a thread in the [Cloudflare Community](https://community.cloudflare.com/) in case of any follow-up questions.
